src/utils/notifications.ts

```diff
@@ -4,28 +4,66 @@ import {
   GetColonyContributorsNotificationDataDocument,
   GetColonyContributorsNotificationDataQuery,
   GetColonyContributorsNotificationDataQueryVariables,
+  GetNotificationUsersDocument,
+  GetNotificationUsersQuery,
+  GetNotificationUsersQueryVariables,
   NotificationsDataFragment,
   NotificationUserFragment,
 } from '~graphql';
 import { getAllPagesOfData, GetDataFn } from './graphql';
 
+export enum NotificationCategory {
+  Mention = 'Mention',
+  Payment = 'Payment',
+  Admin = 'Admin',
+}
+
 export enum NotificationType {
-  Action = 'Action',
+  ExpenditureReadyForReview = 'ExpenditureReadyForReview',
+  ExpenditureReadyForFunding = 'ExpenditureReadyForFunding',
+  ExpenditureReadyForRelease = 'ExpenditureReadyForRelease',
+  ExpenditureFinalized = 'ExpenditureFinalized',
+  ExpenditureCancelled = 'ExpenditureCancelled',
+  PermissionsAction = 'PermissionsAction',
```

Comment:
I am a bit confused why we call it a `PermissionsAction` 😅

Reply:
Haha, it's basically an action made using permissions as the decision method :) If the name is too confusing I'm open to suggestions!